crates/rover-client/src/query/subgraph/fetch.rs

```diff
@@ -63,20 +63,17 @@ fn get_services_from_response_data(
     }
 }
 
-fn get_sdl_for_service(
-    services: ServiceList,
-    service_name: &str,
-) -> Result<String, RoverClientError> {
+fn get_sdl_for_service(services: ServiceList, name: &str) -> Result<String, RoverClientError> {
     // find the right service by name
-    let service = services.iter().find(|svc| svc.name == service_name);
+    let service = services.iter().find(|svc| svc.name == name);
```

Comment:
do we wanna update our variable names as well so that it's easier for us to remember to use the new product language?

Reply:
yeah, I think swapping `name` for `subgraph` here would be good! Good idea!